Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,520 @@
+from datetime import datetime
+import typing
+import urllib.parse
+from dateutil import parser
+
+import dateutil
+import requests
+
+# import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+from typing import Dict, Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+XSOAR_TO_ASM_STATUS_MAP = {
+    0: "open_new",
+    1: "open_in_progress",
+    2: "closed",
+    3: "closed"
+}
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def __init__(self,
+                 access_key: str,
+                 secret_key: str,
+                 base_url: str,
+                 verify: bool,
+                 proxy: bool,
+                 timeout: int,
+                 limit: int,
+                 project_id: typing.Optional[int],
+                 collection_ids: typing.Optional[typing.List[str]]):
+        self._access_key: str = access_key
+        self._secret_key: str = secret_key
+        self.base_url: str = base_url
+
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, ok_codes=(200,))
+
+        self.timeout: int = timeout
+        self.limit: int = limit
+        if project_id:
+            self.project_id: Optional[int] = project_id
+        else:
+            self.project_id = None
+
+        if collection_ids:
+            self.collection_ids: typing.List[str] = collection_ids
+        else:
+            self.collection_ids = []
+
+    def make_request(self, method: str, endpoint: str, **kwargs) -> typing.Union[str, dict, bytes]:
+        headers = {
+            "INTRIGUE_ACCESS_KEY": self._access_key,
+            "INTRIGUE_SECRET_KEY": self._secret_key
+        }
+        if kwargs.get('headers') is not None:
+            headers.update(kwargs.pop('headers'))
+
+        url = urljoin(self.base_url, endpoint)
+
+        response = requests.request(method, url, headers=headers, **kwargs)
```

Comment:
why don't you use the `_http_request` method from the `BaseClient` which i see you inherit from?